src/main/java/core/basesyntax/Areacounter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Areacounter {

```

Comment:
в назві класів та інтерфесів кожне нове слово повинно починатись з великої літери
```suggestion
public interface AreaCounter {

```